tensorflow-ops/tests/GradientTest.hs

```diff
@@ -324,6 +324,14 @@ testPad =
     V.fromList [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1] @=? dx
     V.fromList [2, 2, 3] @=? s
 
+testSqrt :: Test
+testSqrt = testCase "testSqrt" $ do
+    [dx] <- TF.runSession $ do
+        x <- TF.render $ TF.vector [0.25 :: Float]
```

Comment:
Could you choose an input where the expected gradient isn't 1 or 0?

Reply:
Yes, I've just fixed it.